python/desc/skycatalogs/skyCatalogs.py

```diff
@@ -359,6 +360,9 @@ def get_objects_by_hp(self, hp, region=None, obj_type_set=None,
         '''
         object_list = ObjectList()
 
+        if hp not in self._hp_info:
```

Comment:
I think you had mentioned something about putting a warning here if a healpixel intersecting the requested region didn't have a corresponding catalog file.  I agree that might be useful info to output here.

Reply:
Ok, I'll add that.